code/modules/mob/living/carbon/human/species_types/squidpeople.dm

```diff
@@ -32,6 +43,71 @@
         . = capitalize(squid_name(genderToFind))
         if(!findname(.))
             break
-			
+
 /datum/species/squid/after_equip_job(datum/job/J, mob/living/carbon/human/H)
-	H.grant_language(/datum/language/rlyehian)	
+	H.grant_language(/datum/language/rlyehian)
+
+	var/datum/action/innate/squid_change/S = new
+	S.Grant(H)
+
+/datum/species/squid/on_species_loss(mob/living/carbon/human/H)
```

Comment:
Put this in here too:
```
var/datum/action/innate/squid_change/S = locate(/datum/action/innate/squid_change) in M.actions
S?.Remove(M)
```

Reply:
Done

Reply:
What is M here? Getting an undefined var

Reply:
I can't find this, reverting for now

Reply:
My bad. `H` not `M`. You should be able to figure this out by now though. And this still needs to be done.